page.go

```diff
@@ -48,8 +54,39 @@ type Page interface {
 	Emoji() string
 }
 
+type Date struct {
```

Comment:
I had to do multiple tries until a date worked, I think we need to support more date formats

Reply:
I will look for a date parse library, which could be more flexible than the go standard library.

Reply:
Hey, i'm back :) i found this libraries that could help with this issue?
* https://github.com/tj/go-naturaldate: this is my favorite 
* https://github.com/markusmobius/go-dateparser: this one seems to be an overkill for this feature
What do you think?